etcdserver/server.go

```diff
@@ -62,7 +61,8 @@ const (
 	StoreKeysPrefix    = "/1"
 
 	purgeFileInterval      = 30 * time.Second
-	monitorVersionInterval = 10 * time.Second
+	monitorVersionInterval = 5 * time.Second
+	versionUpdateTimeout   = 1 * time.Second
```

Comment:
i think they are different things. one is a timeout and another one is a retry interval.